OracleSOASuite/dockerfiles/12.2.1.3/Dockerfile

```diff
@@ -31,20 +32,36 @@ ENV FMW_JAR1=fmw_12.2.1.3.0_soa.jar \
 # -----------------------------------
 ADD  $FMW_JAR1 $FMW_JAR2 /u01/
 COPY container-scripts/* /u01/oracle/dockertools/
-RUN  yum install -y hostname &&    \
-     rm -rf /var/cache/yum &&      \
-     cd /u01 && chmod 755 *.jar && \
-     chmod a+xr /u01/oracle/dockertools/*.*
+RUN  cd /u01 && chmod 755 *.jar && \
+     chmod +xr /u01/oracle/dockertools/*.*
+
 #
+# Install SOA
+# -----------------------------------
 USER oracle
 COPY install/* /u01/
 RUN cd /u01 && \
   $JAVA_HOME/bin/java -jar $FMW_JAR1 -silent -responseFile /u01/soasuite.response -invPtrLoc /u01/oraInst.loc -jreLoc $JAVA_HOME -ignoreSysPrereqs -force -novalidation ORACLE_HOME=$ORACLE_HOME && \
   $JAVA_HOME/bin/java -jar $FMW_JAR2 -silent -responseFile /u01/soasuite.response -invPtrLoc /u01/oraInst.loc -jreLoc $JAVA_HOME -ignoreSysPrereqs -force -novalidation ORACLE_HOME=$ORACLE_HOME INSTALL_TYPE="Service Bus" && \
-  rm -fr /u01/*.jar /u01/*.response /u01/*.loc
+  rm -fr /u01/*.jar /u01/*.response
+
+#
+# Rebuild from base image
+# -----------------------
+FROM oracle/fmw-infrastructure:12.2.1.3
+
```

Comment:
Move the maintainer label here.

Reply:
Samba, add your name as maintainer not Nagaraj as he is not involved in this project.